qt/widgets/common/inc/MantidQtWidgets/Common/QtPropertyBrowser/qtpropertymanager.h

```diff
@@ -877,6 +877,8 @@ QSizeF qBound(QSizeF minVal, QSizeF val, QSizeF maxVal);
 
 template <class Value> static void orderBorders(Value &minVal, Value &maxVal) {
   if (minVal > maxVal)
+    // std:swap cannot replace qSwap for class Value
+    // cppcheck-suppress qSwapCalled
     qSwap(minVal, maxVal);
```

Comment:
```suggestion
  if (minVal > maxVal) {
    using std::swap;
    swap(minVal, maxVal);
  }
```
I think we can call `swap` here as long as we don't directly call `std::swap`, then if `swap` is implemented on a type that version is used instead of the `std::swap` fallback option for built-in types, e.g. [link](https://doc.qt.io/qt-6/qtswap-qtcore-proxy.html)

When I tested it I also added `#include <utility>` at the top of the file.